.gitignore

```diff
@@ -1,3 +1,5 @@
 .idea/
 node_modules/
 coverage/
+dist/
+.~lock*
```

Comment:
When I installed the latest modules it updated the package-lock.json. I've added it to the git ignore for future but I'm not sure how to remove it from the pr. I've also added the dist files as I'm not sure we want to include generated files in prs.